asyncpg/connection.py

```diff
@@ -342,6 +342,7 @@ async def _get_statement(
         *,
         named: bool=False,
         use_cache: bool=True,
+        ignore_custom_codec=False,
```

Comment:
Make sure you include `ignore_custom_codec` in the cache key below so that prepared statements with and without custom codecs in the pipeline don't get confused.

Reply:
Good catch! Will fix, thanks!